sevenseconds/config/securitygroup.py

```diff
@@ -13,8 +13,8 @@ def configure_security_groups(account: object, region: str, trusted_addresses: s
             update_security_group(account.session, region, sg_name, trusted_addresses)
 
 
-def chunks(l, n):
-    """ Yield successive n-sized chunks from l.
+def chunks(list, n):
```

Comment:
I am not sure if I like the change because `list` is a type in python

Reply:
Good point, any suggestion?

Reply:
Changed to collection.